mu-plugins/blocks/screenshot-preview/js/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * WordPress dependencies
+ */
+import { createElement, render } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block.js';
+
+const init = () => {
+	const blockElements = document.getElementsByClassName( 'wporg-screenshot-preview-js' );
+
+	if ( ! blockElements ) {
+		return;
+	}
+
+	for ( let i = 0; i < blockElements.length; i++ ) {
```

Comment:
Any reason not to use `forEach`?

Reply:
What are the arguments for using `forEach`?

Reply:
This pretty mush sums it up https://stackoverflow.com/questions/43031988/javascript-efficiency-for-vs-foreach#answer-43032526

Personally as a dev I much prefer to write and read less code. I don't think the perf benefit with a for loop outweighs this in most cases. I tend to stick to all the modern native array functions: forEach, map, some, reduce